gui/options.cpp

```diff
@@ -2613,10 +2615,12 @@ void GlobalOptionsDialog::apply() {
 	bool isRebuildNeeded = false;
 
 	Common::U32String savePath(_savePath->getLabel());
-	if (!savePath.empty() && (savePath != _("Default")))
+	if (!savePath.empty() && (savePath != _("Default")) && (!savePath.contains(_(" (temporary)"))))
 		ConfMan.set("savepath", savePath.encode(), _domain);
-	else
+	else if (savePath.empty() || (savePath == _("Default"))){
```

Comment:
This looks like you're preventing the path from being edited at all if it was specified in the command line. This should not be the case.

Reply:
the path can be edited. That block just resets the savepath if it's the default string or it's empty.